backend/lcfs/web/api/organization/views.py

```diff
@@ -28,6 +32,65 @@
 get_async_db = dependencies.get_async_db_session
 
 
+@router.get("/export", response_class=StreamingResponse)
+@roles_required("Government")
+async def export(request: Request, db: AsyncSession = Depends(get_async_db)):
+    """
+    Endpoint to export information of all organization
+    """
+    # The export_format can be 'xls', 'xlsx', or 'csv'
+    export_format="xls"
+    media_type = 'application/vnd.ms-excel'
+
+    try:
+        # Fetch all organizations from the database
+        result = await db.execute(
+            select(Organization).options(
+                    joinedload(Organization.org_status)
+                )
+            .order_by(Organization.organization_id)
+        )
+        organizations = result.scalars().all()
+
+        # Prepare data for the spreadsheet
+        data = [
+            [
+                organization.organization_id,
+                organization.name,
+                # TODO: Update this section with actual data retrieval
+                # once the Compliance Units models are implemented.
+                123456,
+                organization.org_status.status.value,
+            ] for organization in organizations
+        ]
+
+        # Create a spreadsheet
+        builder = SpreadsheetBuilder(file_format = export_format)
+
+        builder.add_sheet(
+            sheet_name="Organizations",
+            columns=["ID", "Organization Name", "Compliance Units", "Registered"],
```

Comment:
Just add one more column here called 'In Reserve' so we have it when we get to compliance reporting.

Reply:
Sure, it's added.